presto-docs/src/main/sphinx/release/release-341.md

```diff
@@ -0,0 +1,164 @@
+  ``query.max-scan-physical-bytes`` config property was not defined. ({issue}`5009`)
+* Fix incorrect sampling ratio when `TABLESAMPLE` is used with percentage fraction. ({issue}`5074`)
+* Correct rejecting queries using `TABLESAMPLE` with ratio being not a number. ({issue}`5074`)
+* Fail with an explicit error rather than `OutOfMemoryError` for certain operations. ({issue}`4890`)
+
+## Security Changes
+
+* Add [Salesforce password authentication](/security/salesforce). ({issue}`4372`)
+* Add support for interpolating [secrets](/security/secrets) into access-control.properties. ({issue}`4854`)
+* Only request HTTPS client certificate when certificate authentication is enabled. ({issue}`4804`)
+* Add user mapping configuration to preserve lower case or upper case user names. ({issue}`4736`)
+
+## Web UI
+
+* Fix displaying physical input read time in detailed query view. ({issue}`4962`)
+
+## JDBC Driver Changes
+
+* Implement `ResultSet.getStatement()`. ({issue}`4957`)
+
+## BigQuery Connector Changes
+
+* Add support for HOURLY partitioning of tables. ({issue}`4968`)
+* Redact value of `bigquery.credentials-key` from logs. ({issue}`4968`)
+
+## Cassandra Connector Changes
+
+* Map Cassandra `TIMESTAMP` type to Presto `TIMESTAMP(3) WITH TIME ZONE` type. ({issue}`2269`)
+
+## Hive Connector Changes
+
+* Skip stripes and row group based on timestamp statistics for ORC files. ({issue}`1147`)
+* Skip DeepArchive objects when `skip-glacier-objects` is set. ({issue}`5002`)
+* Use temporary staging directory for write operation to sorted bucketed table. ({issue}`3434`)
+* Invalidate cached information on `GRANT` and `REVOKE`. ({issue}`4768`)
+* Add Parquet and RCBinary [configuration properties](hive_configuration_properties) `hive.parquet.time-zone` and `hive.rcfile.time-zone` to adjust binary timestamp
+  values to a specific time zone. For Hive 3.1+, this should be set to UTC. The default value is the JVM default time zone, for
+  backwards compatibility with earlier versions of Hive. ({issue}`4799`)
+* Add ORC [configuration property](hive_configuration_properties) `hive.orc.time-zone` to set the default time zone for legacy ORC files that did not declare a time
+  zone. ({issue}`4799`)
+* Replace the `hive.time-zone` configuration property with the format specific properties. ({issue}`4799`)
+* Allow to optionally use cluster role in s3 mapping. ({issue}`4931`)
+* Remove support for bucketing on timestamp. ({issue}`4759`)
+* Improve performance of partition fetching from Glue. ({issue}`4938`)
+* Fix invalid timestamp values for nested data in Text, Avro, Sequence File, JSON and CSV formats. ({issue}`4799`)
+* Fix query failure when reading a ORC ACID table with a filter, after the table underwent a minor table compaction. ({issue}`4622`)
+* Fix issue where queries could fail if Rubix caching was used and cached data was evicted during query execution. ({issue}`3580`)
```

Comment:
we decided that the fact Rubix is used is an implementation detail and user should not be concerned, just the same way as we treat Airlift.

Rubix caching -> Storage Caching

This is how we call it @ https://prestosql.io/docs/current/connector/hive-caching.html